pumpkin/src/main.rs

```diff
@@ -193,6 +194,10 @@ async fn main() -> io::Result<()> {
             {
                 let (player, world) = server.add_player(client).await;
                 world.spawn_player(&BASIC_CONFIG, player.clone()).await;
+
+                // inform client about commands so that command suggestions and tab completion work
+                send_c_commands_packet(player.clone(), &server.command_dispatcher).await;
```

Comment:
We definitely don't want to have this in the main. It would be much nicer and make more sense to put this into the Player or World. There would be also no need for cloning then

Reply:
I put it in spawn_player now and removed some .clone()s (tho it's just Arcs so these clones are cheap). I'm pretty sure I put it in the correct place for following vanilla packet order, too. 

And omg why I didn't start using Intellij for browsing vanilla source code earlier and instead relied on text search only. It's soo much easier this way